src/shamrock/src/banner.cpp

```
@@ -0,0 +1,204 @@
+namespace {
+
+    std::optional<std::string> get_banner_pic_path() {
+        constexpr const char *kitty_banner_path
+            = "../doc/mkdocs/docs/assets/no_background_nocolor.png";
+        if (std::filesystem::exists(kitty_banner_path)) {
+            return kitty_banner_path;
+        }
+        return std::nullopt;
+    }
+
+    bool try_print_picture() {
```

Cppcheck (2.21.0)

[style] The function 'try_print_picture' is never used.

(unusedFunction)